Fix cors issue

```
Caused by: java.lang.IllegalArgumentException: When allowCredentials is true, allowedOrigins cannot contain the special value "*"since that cannot be set on the "Access-Control-Allow-Origin" response header. To allow credentials to a set of origins, list them explicitly or consider using "allowedOriginPatterns" instead.
        at org.springframework.web.cors.CorsConfiguration.validateAllowCredentials(CorsConfiguration.java:457) ~[spring-web-5.3.1.jar:5.3.1]
        at org.springframework.web.servlet.handler.AbstractHandlerMapping.getHandler(AbstractHandlerMapping.java:520) ~[spring-webmvc-5.3.1.jar:5.3.1]
        at org.springframework.web.servlet.DispatcherServlet.getHandler(DispatcherServlet.java:1255) ~[spring-webmvc-5.3.1.jar:5.3.1]
        at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1037) ~[spring-webmvc-5.3.1.jar:5.3.1]
        at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:961) ~[spring-webmvc-5.3.1.jar:5.3.1]
        at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006) ~[spring-webmvc-5.3.1.jar:5.3.1]
```

--- a/src/main/java/de/mp/istint/server/config/WebConfig.java
+++ b/src/main/java/de/mp/istint/server/config/WebConfig.java
@@ -5,8 +5,10 @@ import java.util.List;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 import org.springframework.beans.BeansException;
+import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.ApplicationContext;
 import org.springframework.context.ApplicationContextAware;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Profile;
 import org.springframework.http.converter.HttpMessageConverter;
@@ -16,6 +18,8 @@ import org.springframework.web.servlet.config.annotation.CorsRegistry;
 import org.springframework.web.servlet.config.annotation.EnableWebMvc;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
+import de.mp.istint.server.config.data.CorsData;
+
 @Configuration
 @EnableWebMvc
 @Profile({ "dev", "prod" })
@@ -23,6 +27,12 @@ public class WebConfig implements WebMvcConfigurer, ApplicationContextAware {
 
     private final long MAX_AGE_SECS = 3600;
     private ApplicationContext applicationContext;
+
+    @Bean
+    @ConfigurationProperties("istint.cors")
+    public CorsData corsData() {
+        return new CorsData();
+    }
 
     /**
      * Note: The CORS-confgiruation via this method is essential. This setting is handled by Spring
@@ -32,11 +42,12 @@ public class WebConfig implements WebMvcConfigurer, ApplicationContextAware {
      */
     @Override
     public void addCorsMappings(CorsRegistry registry) {
+        CorsData cData = corsData();
         registry.addMapping("/**")
-                .allowedOrigins("*")
+                .allowedOrigins(cData.getAllowedOrigins().toArray(new String[] {}))
                 .allowedMethods("GET", "POST", "PUT", "PATCH", "DELETE", "OPTIONS")
                 .allowedHeaders("*")
-                .allowCredentials(true)
+                .allowCredentials(cData.isAllowCredentials())
                 .maxAge(MAX_AGE_SECS);
     }
 

--- a/src/main/java/de/mp/istint/server/config/data/CorsData.java
+++ b/src/main/java/de/mp/istint/server/config/data/CorsData.java
@@ -1,0 +1,13 @@
+package de.mp.istint.server.config.data;
+
+import java.util.List;
+
+import lombok.Data;
+
+@Data
+public class CorsData {
+    List<String> allowedOrigins;
+    List<String> allowedMethods;
+    List<String> allowedHeaders;
+    boolean allowCredentials;
+}

--- a/src/main/java/de/mp/istint/server/service/EventService.java
+++ b/src/main/java/de/mp/istint/server/service/EventService.java
@@ -34,6 +34,7 @@ public class EventService {
             return eventRepository.save(event);
         } else {
             if (inDb.get().getOwnerId().equals(user.getId())) {
+                event.setOwnerId(user.getId());
                 event.setLastModified(LocalDateTime.now());
                 return eventRepository.save(event);
             }

--- a/src/main/resources/application.yml
+++ b/src/main/resources/application.yml
@@ -19,11 +19,12 @@ logging:
     org.springframework.security.authentication: debug
     org.springframework.security: debug
 
+
 istint:
-  app:
-    piep: pups
-    redirects: 
-      - "a"
-      - "b"
+  cors:
+    allowedOrigins: 
+      - '*'
+    allowCredentials: false
+  
 
 

